.ci/pipeline/ci.yml

```diff
@@ -57,6 +57,57 @@ jobs:
       pip install isort black[jupyter]==24.1.1
       isort --profile black --check . && black --check .
     displayName: 'Linting'
+- job: LinuxCondaRecipe
+  dependsOn: Lint
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Python3.11:
+        PYTHON_VERSION: '3.11'
+  pool:
+    vmImage: 'ubuntu-22.04'
+  steps:
+  - script: |
+      bash .ci/scripts/install_dpcpp.sh
+    displayName: "dpcpp installation"
+  - script: |
+      conda config --add channels conda-forge
+      conda config --set channel_priority strict
+      conda update -y -q --all
+    displayName: "Conda update"
+  - script: |
+      conda create -y -q -n build-env python=$(PYTHON_VERSION) conda-build conda-verify
+    displayName: "Conda install"
```

Comment:
Please address message from the logs.
The build conda recipe pipeline shows warning 
```
Free memory is lower than 5%; Currently used: 95.03%
```